src/app/components/Account/index.tsx

```diff
@@ -88,22 +93,22 @@ export const Account: FC<AccountProps> = ({ account, isLoading, roseFiatValue, s
           </dd>
 
           <dt>{t('common.balance')}</dt>
-          <dd>{t('common.valueInRose', { value: balance })}</dd>
-          {roseFiatValue && balance && (
+          <dd>{t('common.valueInToken', { value: balance, ticker: tickerName })}</dd>
+          {!!tokenFiatValue && balance && (
```

Comment:
!! also returns false when value is 0. We dont care to show 0 USD everywhere on testnet, but I think the abstraction should be clearer about that, instead of treating 0 and undefined the same.

Reply:
OK, so I have added an explicit `isFree` flag to the hook, and we only hide the value when this flag is on (or still loading, etc.) But in any case, a price of 0 is no longer sufficient reason to hide it. (And of course I have enabled the free flag for TEST tokens.)